frontend_addon/{{ cookiecutter.__folder_name }}/Makefile

```diff
@@ -42,7 +41,6 @@ clean: ## Clean environment
 .PHONY: install
 install: ## Installs the add-on in a development environment
 	@echo "$(GREEN)Install$(RESET)"
-	if [ -d $(GIT_FOLDER) ]; then $(PRE_COMMIT) install; else echo "$(RED) Not installing pre-commit$(RESET)";fi
```

Comment:
We could still install our own git pre-commit hook which runs the make commands. But this can happen in a separate PR.

Reply:
No, please. No pre-commit in frontend.